lib/webview-helpers.js

```diff
@@ -16,23 +18,36 @@ let helpers = {};
 // same as chrome-backed webviews)
 // TODO: some of this function belongs in appium-adb
 async function webviewsFromProcs (adb, deviceSocket) {
+  const WEBVIEW_REGEXP = `@?webview_devtools_remote_(\\d+)`;
```

Comment:
Done.

Though IMHO I consider this less readable as the scope of the constant is only the function.